packages/web-app-app-store/src/views/AppList.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="app-list oc-mb-m">
+    <h2 class="oc-mt-rm" v-text="$gettext('App Store')" />
+    <div class="oc-flex oc-flex-middle">
+      <oc-text-input
+        id="apps-filter"
+        v-model.trim="filterTerm"
+        :label="$gettext('Search')"
+        :clear-button-enabled="true"
+        autocomplete="off"
+      />
+    </div>
```

Comment:
We have moved most of our search terms to the right, I'd keep up with that here as well too

Reply:
Doesnt look good, don't do it